vendor/github.com/opencontainers/runc/libcontainer/user/user.go

```diff
@@ -201,7 +201,7 @@ func ParseGroupFilter(r io.Reader, filter func(Group) bool) ([]Group, error) {
 			if err != nil {
 				// We should return no error if EOF is reached
 				// without a match.
-				if err == io.EOF { //nolint:errorlint // comparison with io.EOF is legit, https://github.com/polyfloyd/go-errorlint/pull/12
+				if err == io.EOF {
```

Comment:
Slightly wondering if we need the updated vendor; could this cause issues with consumers of our code? (looks like we don't need the update ourselves, but it could cause issues for others if we "force" them to update).

Reply:
The vendor code doesn't need cherry-picked, but seems probably safe for master (v25)?

Reply:
Yeah, wasn't sure if there were dependencies updated in runc, or other parts of the code. I guess it's fine (mostly tried to keep it at 'minimum required version'